### PR TITLE
Fix Firefox printing only 1 page

### PR DESCRIPTION
**Pull request by bnavigator**
Fix for #3782 

This fixes not only to print all pages in timetable but on all other sites, too.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -59,6 +59,7 @@ Bugfixes
 - Fix sorting book of abstracts by board number (:issue:`3429`, thanks
   :user:`bpedersen2`)
 - Enforce survey submission limit (:issue:`3256`)
+- Fix Firefox printing only 1 page issue (:issue:`3782`)
```
Comment (ThiefMaster) [inline CHANGES.rst]: feel free to credit yourself see (see the "thanks ..."` in some other entries)

File: indico/web/client/styles/base/_layout.scss [hunk 2]
```diff
@@ -9,6 +9,14 @@ body {
     @extend .flexcol;
 }
 
+/* Work arond issues with Firefox which only displays the first page when
+ * printing */
```
Comment (ThiefMaster) [inline indico/web/client/styles/base/_layout.scss]: single line with `//` is fine (or wrap before `*/`)

File: CHANGES.rst (hunk 1 above)
Comment (bnavigator) [reply CHANGES.rst]: Done

File: indico/web/client/styles/base/_layout.scss (hunk 2 above)
Comment (bnavigator) [reply indico/web/client/styles/base/_layout.scss]: I wanted to keep the same style as the comment for the WebKit fix below and not exceed 80 chars per line. See new commit to keep style consistent.

**Comment by ThiefMaster** [comment]
https://github.com/indico/indico/blob/2c3d3892d5e2b962bc990d24368734bc16cd06d6/.sass-lint.yml#L57-L59

;)

Anyway, no big deal, I just find it a bit odd-looking to have a single word in a new line.

File: CHANGES.rst [hunk 3]
```diff
@@ -59,6 +59,8 @@ Bugfixes
 - Fix sorting book of abstracts by board number (:issue:`3429`, thanks
   :user:`bpedersen2`)
 - Enforce survey submission limit (:issue:`3256`)
+- Fix Firefox printing only 1 page issue (:issue:`3782`, thanks
```
Comment (ThiefMaster) [inline CHANGES.rst]: s/1/one/ (or "first"?)

**Comment by panagiotappl** [comment]
Thanks for the PR! 
This fixes the issue of not printing more than 1 pages with Firefox but other issues arise.
 
There is a big white gap between each day and some entries get squeezed at the bottom of the page. This doesn't happen on Chrome. 
I attached an example of an event with many timetable entries. You can check specifically page 3 to see where all the text gets squeezed. 
[firefox-example.pdf](https://github.com/indico/indico/files/3407332/firefox-example.pdf)

**Comment by ThiefMaster** [comment]
I'm going to close this PR for now; feel free to re-open it when addressing the issues @panagiotappl mentioned in her comment.

**Pull request closed without merging**